.github/workflows/manage-runs.yml

```diff
@@ -1,13 +1,12 @@
 name: Manage runs
 on:
-  push:
-    branches:
-      - master
-      - v[0-9]+\.[0-9]+-dev
+  pull_request:
+    types:
+      - closed
```

Comment:
Closed triggered when I merge a PR?

Reply:
Yes, this will trigger on both merge and close actions